conf/app.routes

```diff
@@ -8,6 +8,7 @@ GET        /vat-certificate
 
 GET        /make-payment/:amountInPence/:taxPeriodMonth/:taxPeriodYear/:vatPeriodEnding/:chargeType/:dueDate/:chargeReference        controllers.MakePaymentController.makePayment(amountInPence: Long, taxPeriodMonth: Int, taxPeriodYear: Int, vatPeriodEnding: String, chargeType: String, dueDate: String, chargeReference: String)
 GET        /make-payment/:amountInPence/:chargeType/:dueDate/:chargeReference                                                        controllers.MakePaymentController.makePaymentNoPeriod(amountInPence: Long, chargeType: String, dueDate: String, chargeReference: String)
+GET        /pay-api/view-and-change/vat/                                                                                             controllers.MakePaymentController.makeFullPaymentHandoff
```

Comment:
Any reason why you chose this route name? It's a little misleading as `pay-api` is the context URL for the microservice `pay-api`. Could we instead just use something like `/make-payment` so it's similar to our other payment URLs above?